src/vulkan-renderer/imgui.cpp

```diff
@@ -83,25 +85,39 @@ ImGUIOverlay::ImGUIOverlay(const wrapper::Device &device, const wrapper::Swapcha
     m_push_const_block.translate = glm::vec2(0.0f, 0.0f);
 
     // Load font texture
-    unsigned char *font_texture_data{};
-    int font_texture_width{0};
-    int font_texture_height{0};
 
-    const char *font_filename = "assets/fonts/NotoSans-Bold.ttf";
+    // TODO: Move this data into a container class; have container class also support bold and italic.
+    constexpr char *FONT_FILE_NAME = "assets/fonts/NotoSans-Bold.ttf";
```

Comment:
nit pick, its actually the path not the file name.

Reply:
Fixed.